docs/topics/configmap-example-intro-paragraph.adoc

```diff
@@ -1,3 +1,7 @@
+IMPORTANT
```

Comment:
When marking up admonition blocks in multi-line format, enclose the admonition type in square brackets, otherwise your admonition will be rendered as plain text followed by an delimited block, which looks different form an admonition.

Reply:
This suggestion applies to multiple instances in this diff.